src/views/Header.js

```diff
@@ -71,6 +76,24 @@ class Header extends React.PureComponent<void, HeaderProps, HeaderState> {
     return this._getHeaderTitleString(lastScene);
   }
 
+  _getBackButtonTitleStyle(scene: NavigationScene): ?Style {
+    const lastScene = this._getLastScene(scene);
+    if (!lastScene) {
+      return null;
+    }
+    const {
+      headerBackTitleStyle,
```

Comment:
> True, but the purpose of this change is also that it makes the style of the button match while transitioning from one scene to the other. So IMO it should take the style from the same place as where the text is coming from.

Shouldn't we then assume that style of a back button equals style of a previous scene title?